src/dev-app/slide-toggle/slide-toggle-demo.html

```diff
@@ -1,29 +1,15 @@
 <div class="demo-slide-toggle">
-
-  <mat-slide-toggle color="primary" [(ngModel)]="firstToggle">
-    Default Slide Toggle
-  </mat-slide-toggle>
-
-  <mat-slide-toggle [(ngModel)]="firstToggle" disabled>
-    Disabled Slide Toggle
-  </mat-slide-toggle>
-
-  <mat-slide-toggle [disabled]="firstToggle">
-    Disable Bound
-  </mat-slide-toggle>
+  <mat-slide-toggle color="primary" [(ngModel)]="firstToggle">Default Slide Toggle</mat-slide-toggle>
+  <mat-slide-toggle [(ngModel)]="firstToggle" disabled>Disabled Slide Toggle</mat-slide-toggle>
+  <mat-slide-toggle [disabled]="firstToggle">Disable Bound</mat-slide-toggle>
+  <mat-slide-toggle hideIcon [(ngModel)]="firstToggle">No icon</mat-slide-toggle>
```

Comment:
Wondering if there could be a better name. Like "hide indicators" or something else

Reply:
It would be nice to align it with the naming in `MatOption.hideSingleSelectionIndicator`, but I'm not sure if "indicator" is descriptive enough.